default/scripting/macros/named_values.focs.txt

```diff
@@ -66,4 +69,4 @@ NamedReal name = "FU_RAMSCOOP_REFUEL" value = 0.4
 
 #include "/scripting/macros/misc.macros"
 #include "/scripting/macros/base_prod.macros"
-#include "/scripting/macros/priorities.macros"
\ No newline at end of file
+#include "/scripting/macros/priorities.macros"
```

Comment:
Did something change here?

Reply:
newline at EOF got missing

Reply:
Could you fix it so we merge it.

Reply:
I'd say if it's not automated, it's not that important.  Let's merge it and fix it later.  WDYT?